config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
-end
+  resources :scientists, only: [:show] do
+    resources :experiments, only: [:index, :show] do
+      member do
+        delete 'remove', to: 'scientist_experiments#destroy'
+      end
```

Comment:
This internal member route is not what you want. You only need the nested resource routing. The controller and action are correct, however.